vagrant/Vagrantfile

```diff
@@ -0,0 +1,24 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby et ts=2 sts=2 sw=2 :
+
+nodes = [
+  { :hostname => "centos8", :box => "centos/8" },
+  { :hostname => "centos7", :box => "centos/7" },
+]
+
+Vagrant.configure("2") do |config|
```

Comment:
Don't we need `cpu_mode` configuration to make it run on CentOS CI? Are there any other missing configuration needed for CentOS CI?

Reply:
Probably yes, good point. I didn't run in on centos ci yet.

Reply:
updated